torch/csrc/jit/tracer.h

```diff
@@ -308,20 +308,22 @@ TORCH_API void ensureUniqueIfOutOfPlaced(
     const char* name,
     const at::Tensor& tensor);
 
+TORCH_API void addOutput(Node* node, const at::Tensor& tensor);
+TORCH_API void setOutput(Value* value, const at::Tensor& output);
+TORCH_API void addOutput(Node* node, const std::vector<at::Tensor>& list);
+TORCH_API void addOutput(Node* node, const c10::List<at::Tensor>& list);
 template <
     typename T,
     typename = torch::enable_if_t<
         (!std::is_convertible<torch::decay_t<T>, at::TensorList>::value &&
+         !std::is_convertible<torch::decay_t<T>, c10::List<at::Tensor>>::value &&
```

Comment:
same, at::TensorList should be the same as c10::List<at::Tensor>

Reply:
Right, it should work. I think the problem was that we provided implementation for std::vector<...> instead of c10::List<...>.